app/controllers/user_sessions_controller.rb

```diff
@@ -7,15 +7,15 @@ def new
 
   def create
     if @user = login(params[:email], params[:password])
-      redirect_back_or_to(new_review_path, success: 'Login successful')
+      redirect_back_or_to(new_review_path, success: t('auth.login.ok'))
     else
-      flash.now[:warning] = 'Login failed'
+      flash.now[:warning] = t('auth.login.failed')
       render action: 'new'
     end
   end
 
   def destroy
     logout
-    redirect_to(root_path, info: 'Logged out!')
+    redirect_to(root_path, info: t('auth.login.out'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.